packages/frontend/src/components/SimpleTable/index.tsx

```diff
@@ -0,0 +1,118 @@
+import { Button, Colors, HTMLTable } from '@blueprintjs/core';
+import { friendlyName } from 'common';
+import React, { FC, useEffect, useState } from 'react';
+import { TrackSection } from '../../providers/TrackingProvider';
+import { SectionName } from '../../types/Events';
+import { TableHeader, TableInnerWrapper, TableWrapper } from './SimpleTable';
+
+interface Props {
+    data: any;
+}
+const SimpleTable: FC<Props> = ({ data }) => {
+    const [paginatedData, setPaginatedData] = useState(data.slice(0, 25));
+
+    const [currentPage, setCurrentPage] = useState<number>(0);
+    const pageCount = Math.ceil(data?.length / 25);
+
+    const canNextPage =
+        currentPage + 1 < pageCount && currentPage + 1 !== pageCount;
+    const canPrevPage = currentPage > 0;
+
+    const headerGroup = paginatedData.map((item: {}) => Object.keys(item))[0];
+    const tableRows = paginatedData.map((row: {}) => Object.values(row));
+
+    const modifiedItem = (item: string | boolean) => {
+        if (typeof item === 'boolean') {
+            return item ? 'Yes' : 'No';
+        }
+        return item;
+    };
+
+    useEffect(() => {
+        setPaginatedData(data.slice(currentPage * 25, currentPage * 25 + 25));
+    }, [currentPage]);
+
+    return (
+        <TrackSection name={SectionName.RESULTS_TABLE}>
```

Comment:
@ZeRego do we need to track this section? 

If should change the name @nathaliabj to distinguish this table from the explorer table. Maybe it could be `CHART_VISUALIZATION_TABLE`